packaging/windows/sharedframework/sharedframework.wxs

```diff
@@ -11,6 +11,27 @@
         <Feature Id="MainFeature" Title="Main Feature" Level="1">
             <ComponentGroupRef Id="InstallFiles" />
             <ComponentGroupRef Id="AuthoredRegistryKeys"/>
+            <Component Id="BreadcrumbStoreFolder" Directory="BreadcrumbStore" Guid="{DA957490-EA59-42B9-B9D6-3E80F6FB0ED0}">
```

Comment:
Why not?  It has a well defined machine-wide directory as keypath.  That should be a sufficient seed.  Then again its not really a component where you care about having a stable GUID since it is fire-and-forget.